From bd604dc8daa70b24cb37ee6aac51f58f15f29a5f Mon Sep 17 00:00:00 2001
From: saikat0511 <57610828+saikat0511@users.noreply.github.com>
Date: Tue, 16 May 2023 02:28:50 +0530
Subject: [PATCH] hwdb: fix keyboard entry for IdeapadFlex5 (#27643)

Fixes a bug caused by 19db450f3a243fcaf0949beebafc3025f8e3a98e (#27211).

Also this makes the model more specific.

(cherry picked from commit 738a195bd59dc2e85c15382d17d391c1781aaa4e)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 5d34e02607..60c8e2edf7 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -1007,7 +1007,7 @@ evdev:atkbd:dmi:bvn*:bvr*:svnLENOVO*:pn*IdeaPad*Z370*:pvr*
  KEYBOARD_KEY_ae=!volumedown
  KEYBOARD_KEY_b0=!volumeup
 
-evdev:atkbd:dmi:bvn*:bvr*:svnLENOVO*:pn*IdeaPadFlex5*:pvr*
+evdev:atkbd:dmi:bvn*:bvr*:svnLENOVO:*:pvrIdeaPadFlex514IIL05:*
  KEYBOARD_KEY_a0=!mute
  KEYBOARD_KEY_ae=!volumedown
  KEYBOARD_KEY_b0=!volumeup
